README.md

```diff
@@ -41,6 +40,11 @@ This is used in the [winit](#winit) example for the SVG rendering.
 
 A separate integration for playing Lottie animations is available through the [`velato`](https://github.com/linebender/velato) crate.
 
+### Community
+
+- [bevy-vello](https://github.com/vectorgameexperts/bevy-vello): A bevy integration to render lottie or svg files
```

Comment:
I'm not completely opposed to this, although I think it should be a different PR.

We get into a bit of a challenge of which things do we mention here, and what happens if they get out of date, for example - this might be better as an external page, like the Bevy examples page

I think it's a confused message to have both Velato and Velottie here - I'm not sure what we can do about that in this pr discussion though

Reply:
I'll remove it